kuma/users/admin.py

```diff
@@ -23,20 +22,29 @@ class UserAdmin(BaseUserAdmin):
     Extends the admin view of users to show date_joined field
     add a filter on the field too
     """
-    list_display = ('username', 'fullname', 'email', 'title', 'organization',
-                    'location', 'content_flagging_email', 'tags',
+    list_display = ('username', 'fullname', 'email',
+                    'bio', 'website', 'revisions',
                     'date_joined', 'is_staff', 'is_active')
-    list_editable = ('content_flagging_email', 'tags')
     list_filter = ('is_staff', 'is_superuser', 'is_active', 'date_joined')
     ordering = ('-date_joined',)
     search_fields = ('username', 'homepage', 'title', 'fullname',
                      'organization', 'location', 'bio', 'email', 'tags__name')
 
-    formfield_overrides = {
-        NamespacedTaggableManager: {
-            'widget': TagWidget(attrs={'size': 45})
-        }
-    }
+    def revisions(self, obj):
+        """HTML link to user's revisions with count"""
+        link = urlparams(reverse('dashboards.revisions'),
+                         user=obj.username)
+        count = obj.created_revisions.count()
```

Comment:
The count is definitely needed, at least for counts < 25 or so. If you know off the top of your head a way to do this more quickly I'm all ears; otherwise I can dig around for alternatives.